### PR TITLE
Update SIW to version 11.4.2.20210602

### PR DESCRIPTION
**Pull request by gtopala**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16358)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm getting an error on my device:
image

**Comment by gtopala** [comment]
I don’t know why you cannot download https://siw64.com/download/siwtrial-setup.exe
The link works from a browser of from wget

***@***.***D75ACF.2254C570]

Thank you,
Gabriel Topala

File: manifests/t/TopalaSoftwareSolutions/SIW/2021.06.02.2/TopalaSoftwareSolutions.SIW.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+PackageIdentifier: TopalaSoftwareSolutions.SIW
+Publisher: Topala Software Solutions
+PackageName: System Information for Windows
```
Comment (ItzLevvie) [inline manifests/t/TopalaSoftwareSolutions/SIW/2021.06.02.2/TopalaSoftwareSolutions.SIW.yaml:3]:
```suggestion
PackageName: SIW 2021 Trial
```
image

File: manifests/t/TopalaSoftwareSolutions/SIW/2021.06.02.2/TopalaSoftwareSolutions.SIW.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Moniker: siw
+MinimumOSVersion: 6.1.0.0
+PackageVersion: 2021.06.02.2
```
Comment (ItzLevvie) [inline manifests/t/TopalaSoftwareSolutions/SIW/2021.06.02.2/TopalaSoftwareSolutions.SIW.yaml:6]:
```suggestion
PackageVersion: 2021.06.02b
```
image

File: manifests/t/TopalaSoftwareSolutions/SIW/2021.06.02.2/TopalaSoftwareSolutions.SIW.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ShortDescription: System Information for Windows - Everything you want to know about your computer
+License: Commercial
+LicenseUrl: https://www.gtopala.com/agreement/
+PackageUrl: https://www.gtopala.com/
+Installers:
+- Architecture: neutral
```
Comment (ItzLevvie) [inline manifests/t/TopalaSoftwareSolutions/SIW/2021.06.02.2/TopalaSoftwareSolutions.SIW.yaml:12]:
```suggestion
- Architecture: x86
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

**Pull request closed without merging**